packages/neuron-ui/src/exceptions/index.ts

```diff
@@ -0,0 +1,66 @@
+import { ErrorCode } from 'utils/const'
+
+export class FieldInvalidException extends Error {
+  public code = ErrorCode.FieldInvalid
+  public i18n: {
+    fieldName: string
+  }
+
+  constructor(fieldName: string) {
+    super(`messages.codes.${ErrorCode.FieldInvalid}`)
```

Comment:
```suggestion
    super(`messages.codes.${this.code}`)
```

Reply:
`this` should not be used until `super` is called.

Reply:
ok